src/Terminal/HelpText.ts

```diff
@@ -417,17 +418,17 @@ export const HelpTexts: Record<string, string[]> = {
     "Note that if you use rm to remove a file, the contents of the file will be lost. This is irreversible.",
   ],
   run: [
-    "Usage: run [file name] [-t num_threads] [--tail] [--ram-override ram_in_GBs] [args...]",
+    "Usage: run [file name] [-t num_threads] [--tail] [--ram-override ram_in_GBs] [--temporary] [args...]",
     " ",
     "Execute a program, script or coding contract.",
     " ",
-    "The '[-t num_threads]', '[--tail]', `[--ram-override ram_in_GBs]`, and '[args...]' arguments are only valid",
-    "when running a script. The '-t' flag is used to indicate that the script should be run with the specified",
+    "The '[-t num_threads]', '[--tail]', `[--ram-override ram_in_GBs]`, [--temporary], and '[args...]' arguments are",
+    "only valid when running a script. The '-t' flag is used to indicate that the script should be run with the specified",
     "number of threads. If the flag is omitted, then the script will be run with a single thread by default. The",
-    "'--tail' flag is used to immediately open a tail window for the script being ran. And the '--ram-override'",
-    "flag is used to override the amount of ram (per thread) the script is ran with. If the script ends up using",
-    "more than that amount of ram it will crash. If any of the flags are used, then they MUST come immediately",
-    "after the script name.",
+    "'--tail' flag is used to immediately open a tail window for the script being ran. The '--ram-override' flag is used",
+    "to override the amount of ram (per thread) the script is ran with. If the script ends up using more than that",
+    "amount of ram it will crash. The '--temporary' flag is used to indicate that the script should be excluded from the",
+    "save data. If any of the flags are used, then they MUST come immediately after the script name.",
```

Comment:
From my testing, I don't believe this part is actually true.

Optional: Mention `--` to end special args processing, if someone wants to explicitly pass one of these flags to their script.